packages/nitro-protocol/contracts/NitroAdjudicator.sol

```diff
@@ -129,4 +129,15 @@ contract NitroAdjudicator is ForceMove, MultiAssetHolder {
     ) public pure returns (bool) {
         return _requireValidTransition(nParticipants, isFinalAB, ab, turnNumB, appDefinition);
     }
+
+    /**
+     * @notice Executes an exit by paying out assets and calling external contracts
+     * @dev Executes an exit by paying out assets and calling external contracts
+     * @param exit The exit to be paid out.
+     */
+    function executeExit(Outcome.SingleAssetExit[] memory exit) internal {
+        for (uint256 assetIndex = 0; assetIndex < exit.length; assetIndex++) {
+            executeSingleAssetExit(exit[assetIndex]);
+        }
+    }
```

Comment:
two nits:

For consistency:
```suggestion
    function _executeExit(Outcome.SingleAssetExit[] memory exit) internal {
        for (uint256 assetIndex = 0; assetIndex < exit.length; assetIndex++) {
            _executeSingleAssetExit(exit[assetIndex]);
        }
    }
```
And a question: is this function better grouped with executeSingleAssetExit in MAH? Maybe not - the current location is better at minimizing API size along the chain.